sklearn/metrics/pairwise.py

```diff
@@ -1566,6 +1566,13 @@ def additive_chi2_kernel(X, Y=None):
     return result
 
 
+@validate_params(
+    {
+        "X": ["array-like", "sparse matrix"],
+        "Y": ["array-like", "sparse matrix", None],
```

Comment:
Actually, we should not support sparse matrices since `additive_chi2_kernel` does not support it. We should update the documentation and change `ndarray` to `array-like` for `Y`.

Reply:
I open https://github.com/scikit-learn/scikit-learn/pull/26178 to make sure that any function calling `additive_chi2_kernel` would raise an error with sparse data.

Reply:
I don't understand the purpose of #26178, since it already raises on sparse data